localstack-core/localstack/services/s3/storage/ephemeral.py

```diff
@@ -334,9 +334,10 @@ def copy_from_object(
         :param range_data: the range data from which the S3Part will copy its data.
         :return: the EphemeralS3StoredObject representing the stored part
         """
-        with self._s3_store.open(
-            src_bucket, src_s3_object, mode="r"
-        ) as src_stored_object, self.open(s3_part, mode="w") as stored_part:
+        with (
+            self._s3_store.open(src_bucket, src_s3_object, mode="r") as src_stored_object,
+            self.open(s3_part, mode="w") as stored_part,
+        ):
```

Comment:
I was super curious about this, so here it is for history:
https://www.blog.pythonlibrary.org/2021/09/08/python-3-10-parenthesized-context-managers/

Super neat! 👌